packages/xo-server/src/index.js

```diff
@@ -606,7 +607,21 @@ const setUpConsoleProxy = (webServer, xo) => {
 
         const { remoteAddress } = socket
         log.info(`+ Console proxy (${user.name} - ${remoteAddress})`)
+
+        const data = {
+          timestamp: Date.now(),
+          userId: user.id,
+          userIp: remoteAddress,
+          userName: user.name,
+          vmId: id,
+        }
+        xo.emit('xo:audit', 'consoleOpened', data)
```

Comment:
When will you use 'consoleOpened' parameter ?

Reply:
The `xo:audit` event is handled here https://github.com/vatesfr/xen-orchestra/pull/4789/files#diff-e7393f9074e41d5b19567ef892dfe8fbR77 and `consoleOpened` is passed to https://github.com/vatesfr/xen-orchestra/pull/4789/files#diff-e7393f9074e41d5b19567ef892dfe8fbR114. It's the name of a event.